### DISCUSSION TITLE
Code for Multi RGB LED

### DISCUSSION THREAD

**Discussion by florianHK** [HomeSpan/HomeSpan#128] [Q&A] [answered]
I try to use the RGB example for a multi RGB LED setup. Can anybody tell me how I can add 3 more RGB LEDs to the code. I deleted the 2 other LEDs but don't know how to add a second or more RGB LEDs to the code.

Goal is to create a "homekit status LED dashboard" to visually monitor door locks, garage door etc.

Would be great if somebody could help.

Thanks
Florian

**Comment by HomeSpan** [comment]
Simply duplicate the code that creates the RGB LED for as many copies as you need, remembering to select whatever pins you are going to use to attach your LEDs.  Example:

```C++
  new SpanAccessory();                                                          
    new DEV_Identify("RGB LED","HomeSpan","123-ABC","20mA LED","0.9",0);
    new DEV_RgbLED(32,22,23);                                                      // Create an RGB LED attached to pins 32,22,23 (for R, G, and B LED anodes)

  new SpanAccessory();                                                          
    new DEV_Identify("RGB LED","HomeSpan","123-ABC","20mA LED","0.9",0);
    new DEV_RgbLED(16,17,18);                                                      // Create an RGB LED attached to pins 16,17,18 (for R, G, and B LED anodes)
```

**Comment by florianHK** [comment, marked as answer]
Thank you very much, it works great !!!
